### ISSUE TITLE
High CPU usage on release profile

### ISSUE THREAD

**Issue by KarlitosVII** [imgui-rs/imgui-rs#478]
**Describe the bug**
High CPU usage on release profile. From < 3% in debug to > 30% in release

**To Reproduce**
cargo run --example hello_world --release

**Please describe your environment**
 - Latest git commit so a78fb454973a2546bde42f6d236448fe6d36ab64
 - Windows 10 with latest stable msvc toolchain (1.51.0)

**Other information**
It seems that the culprit is `debug_assertion = false` so for now I have to enable it in release mode which is not that great but it works.
Same bug in my project when using backend from example directory.

**Comment by thomcc** [comment]
Wait, you're seeing that `debug_assertions = true` has lower CPU usage than `debug_assertions = false`?

**Comment by KarlitosVII** [comment]
Yep it seems that it use a full logical cpu usage so between 25% and 35% (I have 4 cpu cores).

**Comment by KarlitosVII** [comment]
The culprit is here https://github.com/glium/glium/blob/master/src/context/mod.rs#L813-L819 so not directly imgui-rs.
With `DebugCallbackBehavior::DebugMessageOnError`-> no problem.
With `DebugCallbackBehavior::Ignore` -> high cpu usage.

**Comment by thomcc** [comment]
Weird, I think you should file the bug with glium then, since that sounds like a bug on their end.
